Solutions/Authomize/Hunting queries/Chain_of_3_or_more_roles.yaml

```diff
@@ -0,0 +1,37 @@
+id: bf03796a-3ed7-440f-bfc3-0c702cf762a9
+name: Chain of 3 or more roles
+description: |
+  'detects chains of more than 3 roles in the account, this is a misconfiguration that can enable lateral movement.'
+requiredDataConnectors:
+  - connectorId: Authomize
+    dataTypes:
+      - Authomize_v2_CL
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1089
+query: |
+  Authomize_v2_CL
```

Comment:
same comments as mentioned in analytics rule

Reply:
done